src/lib/history.ts

```diff
@@ -0,0 +1,28 @@
+import { WeeklyArticleLog, WeeklyArticleHistory } from '../components/ContributionsEpicTypes';
```

Comment:
Not for this PR I don't think, but it feels slightly odd that we're pulling in types for this pure business logic module from `components`. Maybe we should think about moving those types at some point 🤔

Reply:
True, I think some types will have to be abstracted out of the components folder, but agree we'll need to tackle that separately.